src/main/java/com/dataloom/data/DataApi.java

```diff
@@ -23,19 +30,27 @@
     String GET_DATA_PATH = "getData";
 
     String SET_ID        = "setId";
-    String SYNC_ID        = "syncId";
+    String SYNC_ID       = "syncId";
     String SET_ID_PATH   = "{" + SET_ID + "}";
-    String SYNC_ID_PATH   = "{" + SYNC_ID + "}";
+    String SYNC_ID_PATH  = "{" + SYNC_ID + "}";
+
+    String FILE_TYPE     = "fileType";
 
     @GET( CONTROLLER + "/" + ENTITY_DATA + "/" + SET_ID_PATH )
-    Iterable<SetMultimap<FullQualifiedName, Object>> getEntitySetData( @Path( SET_ID ) UUID entitySetId );
+    Iterable<SetMultimap<FullQualifiedName, Object>> getEntitySetData(
+            @Path( SET_ID ) UUID entitySetId,
+            @Query( FILE_TYPE ) FileType fileType );
 
     // TODO Should discuss what this path should be
     @POST( CONTROLLER + "/" + ENTITY_DATA + "/" + SET_ID_PATH + "/" + GET_DATA_PATH )
```

Comment:
Prefix ENTITY_DATA w/ `historical'. I would consider a GET with a body here as well since this won't change server state and these values have potential to be larger than URL limitations for IE.